lib/main.js

```diff
@@ -37,6 +37,46 @@ const tabs = require("sdk/tabs");
 const userStorage = require("./userStorage");
 const utils = require("./utils");
 const windows = require('sdk/windows').browserWindows;
+const fileIO = require("sdk/io/file");
+const { setInterval } = require("sdk/timers");
+
+function webexStorageFile() {
+  var fHandle = Cc['@mozilla.org/file/directory_service;1']
+    .getService(Ci.nsIProperties)
+      .get('ProfD', Ci.nsIFile); //get the profile direcotry
+
+  // Get to the new web extensions storage path
+  fHandle.append('browser-extension-data');
+  fHandle.append(self.id);
+  fHandle.append('storage.js');
+  if(!fHandle.exists()) { fHandle.create(0, 0o600); }
+
+  return fHandle;
+}
+
+function migrateData(store){
+  let outfile = webexStorageFile().path;
+  let TextWriter = fileIO.open(outfile, "w");
+  if (!TextWriter.closed) {
+    TextWriter.writeAsync(store, function(){
+    console.log("COPYING PRIVACY BADGER DATA TO WEB EXTENSIONS PATH");
```

Comment:
This callback has an `error` parameter; you should probably collect and check it!